widgetssdk/src/main/java/com/glia/widgets/survey/SurveyController.java

```diff
@@ -39,6 +44,22 @@ public void init(@Nullable Survey survey) {
         setQuestions(survey);
     }
 
+    private boolean isAlreadyInit(@Nullable Survey survey) {
+        if (this.survey == null) {
+            return false;
+        }
+        if (survey == null) {
+            return false;
+        }
```

Comment:
I would suggest to unite these two so that it looked like:
```
if (this.survey == null || survey == null) {
      return false;
}
```

Reply:
done